### ISSUE TITLE
[bug]: CLI errors when using relative path as util alias

### ISSUE THREAD

**Issue by ImPrankster** [shadcn-ui/ui#3854]
### Describe the bug

When using relative path as the alias for util import (I'm using a monorepo, the cn function is under /src on the same level with other components, using relative import will cause apps to not be able to resolve `cn`, so I tried to use `import { cn } from "."`, which works fine for using components) ,but the CLI outputs error:
```
[
  {
    "code": "invalid_type",
    "expected": "string",
    "received": "undefined",
    "path": [
      "resolvedPaths",
      "utils"
    ],
    "message": "Required"
  }
]
```
Seems to be caused by the CLI not be able to resolve the path from "." with tsconfig

Possible fix: adding a option in CLI to opt out of trying to resolve the path

### Affected component/components

CLI

### How to reproduce

Edit the component.json > aliases > utils to "."
Try shadcn-ui add

### System Info

```bash
on latest shadcn-ui CLI version
```

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues

**Comment by Antonytm** [comment]
I have the same issue

**Comment by mishelen** [comment]
definitely, there should be clear way described in the docs for projects without aliases

**Comment by nikachello** [comment]
i am already tired with installing this shadcn, documentation is not working for vite/react/ts. does someone have way to teach how can i do it?

**Comment by PradalCyril** [comment]
> i am already tired with installing this shadcn, documentation is not working for vite/react/ts. does someone have way to teach how can i do it?

Hello, I experiment this issue.. and i resolved it with the components.json file and tsconfig. (I'm using vite without path alias... )

I try to put "./" in alias and bim! error : 
` "aliases": {
    "components": "./lib/components",
    "utils": "./lib/utils"
  }`
  
  I replace './' to :
  `"aliases": {
    "components": "@/components",
    "utils": "@/utils"
  }` 
  
  and add this in tsconfig: 
   
  ` "baseUrl": ".",
    "paths": {
      "@/*": ["lib/*"]
    }`
  
  And Done 🟢

**Comment by shadcn** [comment]
This issue has been automatically marked as stale due to one year of inactivity. It will be closed in 7 days unless there’s further input. If you believe this issue is still relevant, please leave a comment or provide updated details. Thank you. (This is an automated message)

**Comment by shadcn** [comment]
This issue has been automatically closed due to one year of inactivity. If you’re still experiencing a similar problem or have additional details to share, please open a new issue following our current issue template. Your updated report helps us investigate and address concerns more efficiently. Thank you for your understanding! (This is an automated message)
